backend/app/api/api.py

```diff
@@ -16,26 +16,26 @@
 router = APIRouter()
 
 # OCP endpoints
-router.include_router(ocpJobs.router, tags=['ocp'])
-router.include_router(results.router, tags=['ocp'])
-router.include_router(graph.router, tags=['ocp.graphs'])
+router.include_router(ocpJobs.router, tags=["ocp"])
+router.include_router(results.router, tags=["ocp"])
+router.include_router(graph.router, tags=["ocp.graphs"])
```

Comment:
As we discussed earlier -- I really think we ought to format everything with Prettier/black for consistency, but *not* in a commit (or ideally PR) where there are "real" changes. That is, we should be able to review the reformatting with a quick glance to confirm there are no breaking changes.

It's fine to do the reformatting in a new PR and then rebase this on top after it's done -- but that still makes *this* PR harder to review than necessary, and it's not the ideal solution.

Reply:
Raised the PR for Formatting the .py files [#139 ](https://github.com/cloud-bulldozer/cpt-dashboard/pull/139)